oat-sa/extension-tao-proctoring
# Fix/ACTP-224/Translate english on spanish interface

Related to: [https://oat-sa.atlassian.net/browse/ACTP-224](https://oat-sa.atlassian.net/browse/ACTP-224)

Added translations for English messages to Spanish.

#### How to test
 
Precondition:
* Change proctor language interface to `Mexican Spanish`;
* You need to have some test-takers on proctor screen;
* Use the following command to update translations: `php tao/scripts/taoUpdate.php`.

Log in as proctor.
On the proctor screen select one or multiple test-takers and try to authorize/terminate them.
You will see a model with list of test-takers that will be processed (or not) during this action.
